### PR TITLE
Extract getPrices from SDK to commons

### PR DESCRIPTION
**Pull request by Sajma27**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-commons/test/core/services/TokenPricesService.test.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    const prices = await tokenPricesService.getPrices();
+    const ethPrices = prices['ETH'];
+    expect(ethPrices.ETH).be.eq(1);
+    expect(ethPrices.USD).be.greaterThan(0);
+
+    const daiPrices = prices['DAI'];
+    expect(daiPrices.ETH).be.greaterThan(0);
+    expect(daiPrices.USD).be.greaterThan(0);
+  });
+});
```
Comment (xitronix) [inline universal-login-commons/test/core/services/TokenPricesService.test.ts:22]: mock fetchTokenInfo

File: universal-login-sdk/src/core/observers/PriceObserver.ts [hunk 2]
```diff
@@ -10,10 +9,12 @@ export type OnTokenPricesChange = (data: TokensPrices) => void;
 export class PriceObserver extends ObserverRunner {
   private lastTokenPrices: TokensPrices = {};
   private callbacks: OnTokenPricesChange[] = [];
+  private tokenPricesService: TokenPricesService;
 
   constructor(private tokensDetailsStore: TokensDetailsStore, tick: number) {
```
Comment (xitronix) [inline universal-login-sdk/src/core/observers/PriceObserver.ts:14]: Replace tokensDetailsStore in constructor by tokenPricesService

File: universal-login-sdk/test/mock/PriceObserver.ts [hunk 3]
```diff
@@ -21,7 +22,9 @@ export const createMockedPriceObserver = () => {
   const resetCallCount = () => {
     callCount = 0;
   };
-  const mockedPriceObserver = new PriceObserver({} as TokensDetailsStore, 150);
+  const tokenPricesService = new TokenPricesService();
+  (tokenPricesService as any).getPrices = () => PRICES;
```
Comment (JustynaBroniszewska) [inline universal-login-sdk/test/mock/PriceObserver.ts:26]: Are you sure it should return `PRICES`?

File: universal-login-sdk/src/integration/http/tokensPricesInCurrency.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+const cryptocompare = require('cryptocompare');
+
+export const getEtherPriceInCurrency = async (currency: 'USD' | 'EUR' | 'GBP'): Promise<string> => {
+  const priceInCurrency = await cryptocompare.price('ETH', currency);
+  return priceInCurrency[currency];
+};
```
Comment (xitronix) [inline universal-login-sdk/src/integration/http/tokensPricesInCurrency.ts:6]: Move to TokenPricesService

**Pull request merged**